apps/src/componentLibrary/modal/Modal.tsx

```diff
@@ -0,0 +1,88 @@
+import classnames from 'classnames';
+import React, {HTMLAttributes, ReactNode} from 'react';
+
+import {Button} from '@cdo/apps/componentLibrary/button';
+import CloseButton from '@cdo/apps/componentLibrary/closeButton';
+
+import moduleStyles from './modal.module.scss';
+
+export interface ModalProps extends HTMLAttributes<HTMLDivElement> {
+  /** Modal title */
+  title?: string;
+  /** Modal image url */
+  modalImageUrl?: string;
+  /** Whether to show image as inline element */
+  isImageInline?: boolean;
+  /** Modal content */
+  content?: string | ReactNode;
+  /** Whether to show secondary button */
+  showSecondaryButton?: boolean;
+  /** Custom class name */
+  className?: string;
+  /** Modal onClose handler */
+  onClose?: () => void;
+  /** Modal close button aria label */
+  closeLabel?: string;
+  /** Modal color */
+  color?: 'light' | 'dark';
```

Comment:
Will this be replaced by theme support?

Reply:
In the future - yes. Currently we decided to stick with variant instead of different themes support because of difficulties with adopting dark theme dialog/modal to existing semantic variables. We'll come back to this when we'll have the capacity, but for now for both `Modal` and `Dialog` we'll just have `mode?: 'light' | 'dark'` prop